src/engine/bufferscalers/enginebufferscalerubberband.cpp

```diff
@@ -17,9 +17,13 @@ using RubberBand::RubberBandStretcher;
 
 namespace {
 
+// TODO (XXX): this should be removed. It is only needed to work around
+// a Rubberband 1.3 bug.
```

Comment:
can we ifdef it based on the rubberband version?

Reply:
That suggestion is not trivial to implement so I'd like to avoid it. 
https://github.com/mixxxdj/mixxx/pull/4853#issuecomment-1185082936

Reply:
I was only able to verify that this is not present the newer rubberband version. So I didn't want to introduce regressions based on speculation.